dynamic-config/config-apollo/src/main/java/io/servicecomb/config/client/ApolloConfig.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.servicecomb.config.client;
+
+import com.netflix.config.ConcurrentCompositeConfiguration;
+
+public class ApolloConfig {
+  public static final ApolloConfig INSTANCE = new ApolloConfig();
+
+  private static ConcurrentCompositeConfiguration finalConfig;
+
+  private static final String SERVER_URL_KEY = "apollo.config.serverUri";
+
+  private static final String SERVER_NAMESPACE = "apollo.config.namespace";
+
+  private static final String SERVER_ENV = "apollo.config.env";
+
+  private static final String SERVER_CLUSTERS = "apollo.config.clusters";
+
+  private static final String APOLLO_SERVICE_NAME = "apollo.config.serviceName";
+
+  private static final String TOKEN = "apollo.config.token";
+
+  private static final String REFRESH_INTERVAL = "apollo.config.refreshInterval";
+
+  private static final String FIRST_REFRESH_INTERVAL = "apollo.config.firstRefreshInterval";
+
+  private static final int DEFAULT_REFRESH_INTERVAL = 3;
+
+  private static final int DEFAULT_FIRST_REFRESH_INTERVAL = 0;
+
+  private ApolloConfig() {
+  }
+
+  public static void setConcurrentCompositeConfiguration(ConcurrentCompositeConfiguration config) {
```

Comment:
must be a ConcurrentCompositeConfiguration ?
it's seems that org.apache.commons.configuration.Configuration is enough?

Reply:
It' true that the lightweight Configuration will be enough to hold all the configuration items and. Replaced already.